ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/detail/DetailSteps.java

```diff
@@ -4,27 +4,50 @@
 
 import static com.codeborne.selenide.Selenide.$;
 
+import io.syndesis.qe.pages.customizations.connectors.detail.ApiClientConnectorDetail;
+
 import org.openqa.selenium.By;
 
 import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
 
 import java.util.List;
 
 import cucumber.api.java.en.Then;
 import io.cucumber.datatable.DataTable;
-import io.syndesis.qe.pages.customizations.connectors.detail.ApiClientConnectorDetail;
 
 public class DetailSteps {
 
     private ApiClientConnectorDetail connectorDetailPage = new ApiClientConnectorDetail();
 
-    @Then("^validate connector detail values")
-    public void editProperty(DataTable dataTable) {
+    @Then("^validate connector detail values by id")
+    public void validateById(DataTable dataTable) {
+        this.validateConnectorDetail(dataTable, "id");
+    }
+
+    @Then("^validate connector detail values by data-testid")
```

Comment:
Those two steps could probably be merged into one with a parameter for which selectors are going to be used

Reply:
yes, indeed.